frontend/integration-tests/tests/kubevirt/non-admin.scenario.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-undef */
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { nonAdminSecret, nonAdminProvider } from './utils/mocks';
+import { createResources } from './utils/utils';
+import { KUBEADMIN_IDP, KUBEADMIN_USERNAME, BRIDGE_KUBEADMIN_PASSWORD, BRIDGE_HTPASSWD_IDP, BRIDGE_HTPASSWD_USERNAME, BRIDGE_HTPASSWD_PASSWORD } from './utils/consts';
+import * as loginView from '../../views/login.view';
+import * as crudView from '../../views/crud.view';
+import Wizard from './models/wizard';
+import Yaml from './models/yaml';
+import { logout } from '../../views/kubevirt/login.view';
+import { execSync } from 'child_process';
+
+describe('Test nonadmin user behaviour', () => {
+  const nonAdminNS = `${testName}-nonadmin`;
+
+  const verifyPermissions = async function(ns: string, perm: string) {
+    const wizard = new Wizard();
+    const yaml = new Yaml();
+
+    await browser.get(`${appHost}/k8s/ns/${ns}/virtualmachines`);
+    if (perm === 'noAccess') {
+      await browser.wait(until.presenceOf(crudView.errorMessage));
+      expect(crudView.errorMessage.getText()).toContain('cannot list resource');
+      // TODO: check it can't use vm dialog once BZ1728523 is fixed.
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1728523 is already VERIFIED

Reply:
Yes, will address this TODO once the new build available with this fix available.